docs/pages/build-reference/variables.md

```diff
@@ -241,10 +241,12 @@ The following are two possible alternative approaches, each with different trade
 
 <div style={{marginTop: -20, display: 'block'}} />
 
+> ⚠️  Secrets set on the Expo website or through the EAS CLI will override any env variables with the same name set within your eas.json or app.config.js files. 
```

Comment:
```suggestion
> ⚠️  Secrets set on the Expo website or through the EAS CLI will override any environment variables with the same name set within **eas.json**. 
```

not sure what the original intent was for including app.config.js, you should only set env vars in eas.json. you can read them in app.config.js if you want.